lighthouse-core/audits/is-on-https.js

```diff
@@ -72,6 +72,11 @@ class HTTPS extends Audit {
       extendedInfo: {
         formatter: Formatter.SUPPORTED_FORMATS.URL_LIST,
         value: insecureRecords
+      },
+      details: {
```

Comment:
> I was thinking devtools would replace entire methods (`_renderTable`) with its own implementation

I think that could still happen, unless we bring the devtools table renderer into the report